packages/ignite-cli/src/commands/plugin.js

```diff
@@ -105,29 +106,110 @@ Generally, you would run this from ./YourApp/ignite/plugins/
 Commands:
   ignite plugin help
   ignite plugin new <your-plugin>
-  ignite plugin list (coming soon)
+  ignite plugin list
+  ignite plugin search <search string>
+  ignite plugin info <plugin name>
```

Comment:
I'd like to promote these to `ignite list`, `ignite search`, and `ignite info` just like we already have `ignite add` and `ignite remove`?

Ignite 2 is really all about the plugins.